source/api_cc/include/DeepPot.h

```diff
@@ -427,6 +427,92 @@ class DeepPotModelDevi
   // function used for nborlist copy
   std::vector<std::vector<int> > get_sel() const;
 };
+
+class DeepPotMaskModel: public DeepPot{
+public:
+  /**
+  * @brief DPMaskModel constructor without initialization.
+  **/
+  DeepPotMaskModel (): DeepPot() {}; 
+  ~DeepPotMaskModel();
+  /**
+  * @brief DP constructor with initialization.
+  * @param[in] model The name of the frozen model file.
+  * @param[in] gpu_rank The GPU rank. Default is 0.
+  * @param[in] file_content The content of the model file. If it is not empty, DP will read from the string instead of the file.
+  **/
+  DeepPotMaskModel(const std::string & model, const int & gpu_rank = 0, const std::string & file_content = "");
+  /**
+  * @brief Initialize the DP.
+  * @param[in] model The name of the frozen model file.
+  * @param[in] gpu_rank The GPU rank. Default is 0.
+  * @param[in] file_content The content of the model file. If it is not empty, DP will read from the string instead of the file.
+  **/
+  void init (const std::string & model, const int & gpu_rank = 0, const std::string & file_content = "");
+public:
```

Comment:
duplicate?

Reply:
Sorry. I found that `DeepPotMaskModel` is under developed. That's not finished for inference. I will fix this later.